build.gradle

```diff
@@ -18,7 +18,10 @@ repositories {
 dependencies {
     compile 'org.glassfish.jersey.containers:jersey-container-servlet:2.25.1'
     compile 'org.glassfish.jersey.media:jersey-media-json-jackson:2.25.1'
-    testImplementation 'junit:junit:4.12'
+    implementation 'org.junit.jupiter:junit-jupiter-api:5.5.2'
```

Comment:
use testImplementation

Reply:
use testImplementation('org.junit.jupiter:junit-jupiter:5.5.2'). Since it's only compiled for tests you can use the complete library.